src/Conf.ml

```diff
@@ -152,11 +153,17 @@ end = struct
     else "Cannot be set in attributes."
 
   let generated_choice_doc ~allow_inline ~all ~doc ~section =
+    let open Format in
     let each_opt =
-      let f acc (_, _, d) = Format.sprintf "%s@,%s" acc d in
+      let f acc (_, _, d) = sprintf "%s@,%s" acc d in
       List.fold all ~init:"" ~f
```

Comment:
Re-reading this, I see that if you want, you could use pp_print_list as done in generated_choice_docv.